src/codeu/chat/server/Model.java

```diff
@@ -84,34 +83,45 @@ public int compare(Time a, Time b) {
   private Set<Uuid> admins = new HashSet<>();
   private Map<Uuid, String> passwords = new HashMap<>();
 
-  //map of plays lacking members so any user who decides to join a certain play
-  //will be added as a character in that play
-  private Map<String, PlayInfo> openPlays = new HashMap<>();
-  private Set<String> availablePlays = new HashSet<>();
+  // store from play title to playinfo object, since there could be multiple instances of
+  // the same play in the future
+  private Store<String, PlayInfo> plays = new Store<>(STRING_COMPARE);
```

Comment:
As mentioned before, PlayInfo, like User, ConversationHeader, etc. is a class that holds the state of the state of the application. Hence it should be in chat/common rather than being a Util class.